public/scripts/worktrollyUI.js

```diff
@@ -29,6 +29,14 @@ $("#createNewTask").ready(function() {
     $("#createNewTask").hide(0);
 });
 
+$("#bodyContent").ready(function() {
+    $("#bodyContent").hide(0);
+});
```

Comment:
do we still need body content because i guess it's changed

Reply:
yeah okay i will delete this